movement/sample_data.py

```diff
@@ -237,6 +248,9 @@ def fetch_dataset(
     ----------
     filename : str
         Name of the file to fetch.
+    with_video : bool, optional
```

Comment:
If you like the alternative description from above, then this one should match too.